test/functional/cypress/integration/ecs/clusters_list.spec.js

```diff
@@ -51,4 +53,35 @@ describe('Amazon ECS: aws-prod-ecsdemo cluster', () => {
       .get('a:contains("Rollback")')
       .click();
   });
+
+  it('shows stored instance details view action', () => {
+    cy.visit('#/applications/ecsapp/clusters');
+
+    cy.get('.sub-group:contains("aws-prod-ecsdemo")')
+      .find('.server-group:contains("v000")')
+      .find('.cluster-container')
+      .find('.instances')
+      .find('.instance-group')
+      .get('a[title="f8757e00-184d-4288-b535-4124a739e7be"]')
+      .click();
+
+
+    cy.get('.details-panel > .header')
+      .get('instance-details-header')
+      .get('.header-text')
+      .get('h3:contains("f8757e00-184d-4288-b535-4124a739e7be")');
+
+    cy.get('.details-panel > .content > .collapsible-section').eq(0)
+      .get('dd:contains("2020-07-20 03:41:03 PDT")');
+
+    cy.get('.details-panel > .content > .collapsible-section').eq(0)
+      .get('dd:contains("aws-final-test-v000")');
+
+    cy.get('.details-panel > .content > .collapsible-section').eq(1)
+      .get('span:contains("Up")');
+
+    cy.get('.details-panel > .content > .collapsible-section > .section-heading > .collapsible-heading').eq(2)
+      .click()
```

Comment:
this is still a lot of DOM traveral to the right element. I'd prefer if we did something very explicit, like:
```
cy.get('[data-test-id="SectionHeader.Networking"]').click()
```
...for each element we are checking the contents of. You'll have to add the `data-test-id`s to the actual instance detail page elements, but it will make this much easier to understand and less liable to break b/c of later changes.